OC Share Sheet/ShareViewController.swift

```diff
@@ -123,7 +123,7 @@ fileprivate func > <T : Comparable>(lhs: T?, rhs: T?) -> Bool {
     
     func createCustomInterface(){
         
-        let rightBarButton = UIBarButtonItem (title:NSLocalizedString("upload_label", comment: ""), style: .plain, target: self, action:#selector(ShareViewController.sendTheFilesToOwnCloud))
+        let rightBarButton = UIBarButtonItem (title:NSLocalizedString("upload_label", comment: ""), style: .plain, target: self, action:#selector(ShareViewController.uploadButtonTapped))
```

Comment:
I see that hasn't been considered before in this repo, but would be great if lines were wrapped at 120 characters, so that can be inspected in Github without horizontal scroll.

Just saying.